shop_profile/tests.py

```
@@ -32,6 +35,15 @@ def setUp(self):
             status=True
         )
         self.client = Client()
+    def test_create_user_no_email(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_create_user_no_email`

Add return type annotation: `None`

(ANN201)